### ISSUE TITLE
An Any File Read Vulnerability

### ISSUE THREAD

**Issue by WangYihang** [GitbookIO/gitbook#1885]
Hi , just now I found an any file read vulnerability in your site : https://www.gitbook.com/
which allows attacker to read any file on the server (such as /etc/passwd)
if you want to know the detail about it , please write to me : <wangyihanger@gmail.com>
thank you. @AaronO @jpreynat @SamyPesse @Soreine @Zhouzi

**Comment by AaronO** [comment]
@WangYihang Just sent you an email

**Comment by WangYihang** [comment]
Hi , Aarono, I've replied to your email just now, and in the mail, I described the details of the vulnerability

**Comment by AaronO** [comment]
@WangYihang I just got your email (and read it), you're authorized to publish the details publicly to this thread.

This isn't a "real" vulnerability, since builds on `gitbook.com` happen in isolated LXC environments, thus the `/etc/passwd` file does not contain any sensitive information (there should not be any sensitive information stored on disk in the temporary build environment).

But you're right that this could lead to a security issue if we did have sensitive info on disk (if you do have an exploit which gets sensitive info, that would be accepted).
